libbeat/dashboards/importer.go

```diff
@@ -241,6 +241,9 @@ func (imp Importer) ImportArchive() error {
 			if err != nil {
 				return err
 			}
+		} else {
+			return newErrNotFound("The directory %s does not contain the necessary subdirectory structure "+
+				"defined by setup.dashboards.beat property.", dir)
```

Comment:
Error message description doesn't sound right, but I can't think of anything better right now.